### PR TITLE
incident report for 2018-07-08

### PR DESCRIPTION
**Pull request by minrk**

### CODE REVIEW

**Comment by choldgraf** [comment]
this looks great - anything more to add here @minrk ?

**Comment by minrk** [comment]
These are all my notes. I'm happy for anyone else to add stuff if they think of it, but this is what I have.

**Comment by choldgraf** [comment]
cool - we can spot-check more notes in another PR if need be, thanks for detailing the process!

File: docs/source/incident-reports/2018-07-08-podsnips-aplenty.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+## Lessons learned
+
+### What went well
+
+- mybinder.org-deploy/scripts/delete-pods.py script was useful for bulk deleting older pods to get things back under control.
+- once noticed, cluster returned to healthy within a few minutes.
+
+List of things that went well. For example,
+
+1. We were alerted to the outage by automated bots before it affected users
```
Comment (betatim) [inline docs/source/incident-reports/2018-07-08-podsnips-aplenty.md]: Really? Where do I need to subscribe or signup for this?

**Pull request merged**